src/containers/Transactions/Simple/NFTokenCancelOffer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+import Account from '../../shared/components/Account'
+import { SimpleRow } from '../../shared/components/Transaction/SimpleRow'
+import { Amount } from '../../shared/components/Amount'
+
+export interface Props {
+  data: {
+    instructions: {
+      cancelledOffers: {
+        amount: { currency: string; amount: number; issuer: string }
+        offerID: string
+        tokenID: string
+        offerer: string
+      }[]
+    }
+  }
+}
+
+const NFTokenCancelOffer = (props: Props) => {
```

Comment:
Use TransactionSimpleProps as the type here.  In #278 I add a way to define the types for the instructions.